keras/layers/reshaping/permute.py

```diff
@@ -62,9 +62,11 @@ def __init__(self, dims, **kwargs):
         self.dims = tuple(dims)
         if sorted(dims) != list(range(1, len(dims) + 1)):
             raise ValueError(
-                "Invalid permutation argument `dims` for Permute Layer. "
-                "The set of indices in `dims` must be consecutive and start "
-                f"from 1. Received dims={dims}"
+                "Invalid permutation `dims` for Permute Layer: %s. "
```

Comment:
You're not providing the value for `%s`.
Also, it would be preferable to use an f-string like it was before.

How about:

```python
f"Permute layer received invalid permutation argument dims={dims}. "
```

Reply:
Also, please add a unit test with an `assertRaiseRegex` for this exception.